src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -50,8 +50,6 @@
     <Compile Include="System\Runtime\Versioning\ComponentGuaranteesAttribute.cs" />
     <Compile Include="System\Runtime\Versioning\ResourceConsumptionAttribute.cs" />
     <Compile Include="System\Runtime\Versioning\ComponentGuaranteesOptions.cs" />
-    <Compile Include="System\Runtime\Versioning\ResourceExposureAttribute.cs" />
-    <Compile Include="System\Runtime\Versioning\ResourceScope.cs" />
```

Comment:
Yes but is there any reason we need to remove them from here? In general we are trying to move things out of corelib if possible so we can share.